gh_client.rb

```diff
@@ -53,24 +58,13 @@ def normalize_issues(issues)
 client = Octokit::Client.new(access_token: ENV["GITHUB_PAT"], per_page: 100)
 client.auto_paginate = true
 
-issues_1 = normalize_issues(
```

Comment:
@Reddshift , this query seems like the correct approach instead of the "diff"/delta approach used before. 

One small enhancement that could be made in the future is to accept both mm-dd and m-d format for the date. 

E.g. https://github.com/himaxwell/maxwell/issues?q=is%3Aissue+label%3A%22Bug%22+closed%3A2022-04-01..2022-04-22 works

but 

https://github.com/himaxwell/maxwell/issues?q=is%3Aissue+label%3A%22Bug%22+closed%3A2022-4-1..2022-4-22 does not.

Reply:
Yeah - I just passed on github's limitation of format.  Should be easy enough to turn a single digit input into double.